Externals/CMake/ITK.cmake

```diff
@@ -52,7 +52,7 @@ else()
 endif()
 
 #If using Qt
-if(SV_EXTERNALS_USE_QT)
+if(SV_EXTERNALS_ENABLE_Qt)
```

Comment:
@updega2 - can we stick to all caps?  Is there a reason we use SV_EXTERNALS_ENABLE_Qt instead of just SV_EXTERNALS_ENABLE_QT?  I know we do this for opencascade to be compatible with them, but I prefer consistency.

Reply:
@osmsc This is because we were initially using find_package with Qt and they use Qt5 for their cmake variable names. When we switched to our own, I truncated to just Qt. In the simvascular cmake files (not externals), it is Qt5 because we obviously still use find_package there. So if we make it capital QT, it will be inconsistent with the simvascular source too. What would you like to do?

Reply:
@updega2 - My preference is to keep ours consistent and all caps (i.e. SV_FOO).  These can map to different things for find package (e.g. Qt5), just like what's done in the code below, right?  It's way better for SV_ variables to be consistently named and then map when necessary to package names.  That said, we need to get this committed.  If you prefer to leave it as Qt for now, we can address in a future clean up.